popUp.css

```diff
@@ -0,0 +1,197 @@
+/* pop-up styles */
+
+.popup-container {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  overflow: auto;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: #c1c7d0e5;
+  backdrop-filter: blur(2px);
+}
+
+.main-pu-container {
+  margin: 40px 16px;
+  background-color: white;
+  border-radius: 16px;
+}
+
+.content-container {
+  display: flex;
+  flex-direction: column;
+  margin: 10px 10px;
+}
+
+.heading-btn {
+  display: flex;
+}
+
+.project-title-pu {
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  margin-bottom: 10px;
+}
+
+.close-btn-pu {
+  position: absolute;
+  right: 0;
+  top: 50px;
+  background: transparent;
+  cursor: pointer;
+  width: 40px;
+  height: 50px;
+  border: none;
+  font-weight: lighter;
+  font-size: 20px;
+  margin-right: 20px;
+}
+
+.devs-pu {
+  display: flex;
+  list-style-type: none;
+  gap: 10px;
+}
+
+.dev-pu {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.dev-pu2 {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-img-pu {
+  width: 100%;
+  height: 100%;
+  margin-top: 15px;
+}
+
+.project-info-pu {
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 10px;
+}
+
+.tags-pu {
+  display: flex;
+  list-style-type: none;
+  align-items: baseline;
+  color: #6070ff;
+}
+
+.tag-pu {
+  font-size: 15px;
+  background-color: #ebebff;
+  width: 80px;
+}
+
+.btn-container-pu {
+  display: flex;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.btn-pu {
+  color: #396df2;
+  font-style: normal;
+  text-align: center;
+  font-weight: 500;
+  font-size: 18px;
+  text-decoration: none;
+  border: 1.5px solid #396df2;
+  width: 140px;
+  margin-top: 15px;
+  height: 50px;
+  border-radius: 12px;
+}
+
+.btn-img-pu {
+  margin-left: 10px;
+}
+
+.btn-pu.t {
+  margin-left: 20px;
+}
+
+/* desktop pop-up styles */
+
+@media (min-width: 992px) {
+  .main-pu-container {
+    margin: 40px 75px 40px 25px;
+  }
+
+  .project-title-pu {
+    font-size: 40px;
+  }
+
+  .close-btn-pu {
+    right: 60px;
+  }
+
+  .dev-pu {
+    font-size: 14px;
+  }
+
+  .dev-pu2 {
+    font-size: 14px;
+  }
+
+  .conatiner-pc-pu {
+    display: flex;
+    margin-top: 10px;
+    justify-content: space-between;
+  }
+
+  .img-container-pu {
+    background-color: #e5e5e5;
+    height: 500px;
+    margin-top: 25px;
+    border-radius: 5px;
+  }
+
+  .project-img-pu {
+    margin: 0 60px;
+    width: 90%;
+    height: 100%;
+  }
+
```

Comment:
- [ ] Your popup must follow the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins,...)  kindly make this requested changes: 
- [ ] Your popup image should have 100% width instead of `width: 90%;` and make sure your image is perfectly centering to the center (you need to remove the margin: 0 60px; and instead you can use text-align, margin: 0 auto, justify-content: center....) kindly make this 👍🏻  👨🏻‍🎨
- [ OPTIONAL] It would be better if the background is not scrollable when the popup page is open. you can use the `overflow:hidden;` CSS property to avoid this. 😄 👨🏻‍🎨 🎨 


Your Implementation | Figma Design
:---:|:---:
![23imag](https://user-images.githubusercontent.com/79058364/175263523-cdd0b2b3-2229-4860-8d71-d37e33a09fd0.png) | ![style1](https://user-images.githubusercontent.com/79058364/173584118-d3329953-5244-41d7-8d3e-500b9bc68ada.png)